llvm/lib/Support/Caching.cpp

```diff
@@ -145,7 +145,8 @@ Expected<FileCache> llvm::localCache(const Twine &CacheNameRef,
       // ensures the filesystem isn't mutated until the cache is.
       if (std::error_code EC = sys::fs::create_directories(
               CacheDirectoryPath, /*IgnoreExisting=*/true))
-        return errorCodeToError(EC);
+        return createStringError(EC, Twine("Can't create cache directory: ") +
```

Comment:
Usually error messages are all lower case, I think?

Reply:
(& any test coverage for this?)

Reply:
That might be true for other parts, but this file has all error messages capitalized. I think I'll follow the current convention in the file for now and someone can happily do a NFC for fixing that later.